Samples/Word-Get-Started-With-Dev-Kit/src/taskpane/taskpane.js

```diff
@@ -0,0 +1,30 @@
+/*
+ * Copyright (c) Microsoft Corporation. All rights reserved. Licensed under the MIT license.
+ * See LICENSE in the project root for license information.
+ */
+
+/* global document, Office, Word */
+
+Office.onReady((info) => {
+  if (info.host === Office.HostType.Word) {
+    document.getElementById("sideload-msg").style.display = "none";
+    document.getElementById("app-body").style.display = "flex";
+    document.getElementById("run").onclick = run;
+  }
+});
+
+export async function run() {
+  return Word.run(async (context) => {
+    /**
+     * Insert your Word code here
+     */
+
+    // Insert a paragraph at the end of the document.
+    const paragraph = context.document.body.insertParagraph("Hello World", Word.InsertLocation.end);
+
+    // Change the paragraph color to blue.
+    paragraph.font.color = "blue";
```

Comment:
We don't describe this behavior in the readmes. We should either mention it or remove this line.